hoot-core-test/src/test/cpp/hoot/core/scoring/MapComparatorTest.cpp

```diff
@@ -88,7 +88,7 @@ class MapComparatorTest : public HootTestFixture
     uut.setUseDateTime();
 
     // Change dates...
-    SetTagValueVisitor vtor("source:datetime", "1989-12-13T12:34:56Z");
+    SetTagValueVisitor vtor(MetadataTags::SourceDateTime(), "1989-12-13T12:34:56Z");
```

Comment:
Replaced all occurrences of `source:datetime` and `source:ingest:datetime` with metadata functions through out the code.